src/plugins/vis_type_timeseries/server/lib/vis_data/request_processors/series/date_histogram.js

```diff
@@ -58,12 +58,31 @@ export function dateHistogram(
     };
 
     const overwriteDateHistogramForEntireTimerangeMode = () => {
-      overwrite(doc, `aggs.${series.id}.aggs.timeseries.auto_date_histogram`, {
+      const metricAggs = Object.values(METRIC_AGGREGATIONS);
+      isAggSupported(series.metrics);
+
+      if (series.metrics.every((metric) => metricAggs.includes(metric.type))) {
```

Comment:
please move to constrant or add comment for `series.metrics.every((metric) => metricAggs.includes(metric.type)`

Reply:
added comment